src/main/java/com/openshift/jenkins/plugins/pipeline/MessageConstants.java

```diff
@@ -0,0 +1,130 @@
+public static final String EXIT_BUILD_NO_POD_OBJ = "\n\nExiting \"" + OpenShiftBuilder.DISPLAY_NAME + "\" unsuccessfully; the build pod for build \"%s\" was not found in time.";
+public static final String EXIT_BUILD_NO_BUILD_CONFIG_OBJ = "\n\nExiting \"" + OpenShiftBuilder.DISPLAY_NAME + "\" unsuccessfully; the build config \"%s\" could not be read.";
+
+/*
+ * These messages are for the "Verify OpenShift Builds" jenkins build step implemented by OpenShiftBuildVerifier
+ * Reminder - "Verify OpenShift Builds" looks at builds started externally from the Jenkins project this step is in 
+ */
+public static final String WAITING_ON_BUILD_STARTED_ELSEWHERE = "  Waiting on build \"%s\" to complete ...";
+public static final String WAITING_ON_BUILD_STARTED_ELSEWHERE_PLUS_DEPLOY = "  Waiting on build \"%s\" to complete followed by a new deployment ...";
+
+
+/*
+ * These messages are for the "Cancel OpenShift Builds" jenkins post-build action implemented by OpenShiftBuildCanceller
+ */
+public static final String CANCELLED_BUILD = "  Cancelled build \"%s\".";
+public static final String EXIT_BUILD_CANCEL = "\n\nExiting \"%s\" successfully with %d builds cancelled.";
+
+/*
+ * These messages are for the "Create OpenShift Resource(s)" jenkins build step implemented by OpenShiftCreator
+ */
+public static final String START_CREATE_OBJS = "\n\nStarting \"" + OpenShiftCreator.DISPLAY_NAME + "\" with the project \"%s\".";
+public static final String CREATED_OBJ = "  Created a \"%s\"";
+public static final String FAILED_OBJ = "  Failed to create a \"%s\"";
+public static final String EXIT_CREATE_BAD = "\n\nExiting \"" + OpenShiftCreator.DISPLAY_NAME + "\" unsuccessfully, with %d resource(s) created and %d failed attempt(s).";
+public static final String EXIT_CREATE_GOOD = "\n\nExiting \"" + OpenShiftCreator.DISPLAY_NAME + "\" successfully, with %d resource(s) created.";
+public static final String TYPE_NOT_SUPPORTED = "  The API resource \"%s\" is not currently supported by this step.";
+
+/*
+ * These messages are shared between "Cancel OpenShift Deployment", "Scale OpenShift Deployment", "Verify OpenShift Deployment", and 
+ * "Trigger OpenShift Deployment", implemented respectively by OpenShiftDeployCanceller, OpenShiftScaler, OpenShiftDeploymentVerifier, and OpenShiftDeployer
+ * 
+ */
+public static final String EXIT_DEPLOY_RELATED_PLUGINS_NO_CFG = "\n\nExiting \"%s\" unsuccessfully; the deployment config \"%s\" could not be read.";
+public static final String START_DEPLOY_RELATED_PLUGINS = "\n\nStarting \"%s\" with deployment config \"%s\" from the project \"%s\".";
+public static final String EXIT_DEPLOY_RELATED_PLUGINS_BAD = "\n\nExiting \"%s\" unsuccessfully; deployment \"%s\" has completed with status:  [%s].";
+public static final String EXIT_DEPLOY_RELATED_PLUGINS_GOOD_REPLICAS_IGNORED = "\n\nExiting \"%s\" successfully; deployment \"%s\" has completed with status:  [Complete].";
```

Comment:
Yep, the first start and exit are shared across those 4, but the two you noted are in fact only used in the "Trigger OpenShift Deployment" and "Verify OpenShift Deployment" steps.

How about I leave the constant names the same (since they apply to more than one step), but break them out into a separate section with a comment that just cites those two?